farm/modeling/predictions.py

```diff
@@ -177,6 +177,10 @@ def _span_to_string(self, token_offsets: List[int], clear_text: str):
             end_ch = len(clear_text)
         else:
             end_ch = token_offsets[end_t]
+
+        final_text = clear_text[start_ch: end_ch].strip()
+        end_ch = start_ch + len(final_text)
+
         return clear_text[start_ch: end_ch].strip(), start_ch, end_ch
```

Comment:
Shall we change the return then to 
`return final_text, start_ch, end_ch` ?

Reply:
Yes, definitely